ftw/blueprints/sections/multilingual.py

```diff
@@ -1,11 +1,11 @@
 from collective.transmogrifier.interfaces import ISection
 from collective.transmogrifier.interfaces import ISectionBlueprint
 from collective.transmogrifier.utils import defaultMatcher
-from plone.multilingual.interfaces import ILanguage
-from plone.multilingual.interfaces import IMutableTG
-from plone.multilingual.interfaces import ITranslatable
-from plone.multilingual.interfaces import ITranslationManager
+from plone.app.multilingual.interfaces import IMutableTG
```

Comment:
Cool.
I think we could just check the Plone version:
```python
from Products.CMFPlone.utils import getFSVersionTuple

if getFSVersionTuple() >= (5,):
    # Plone 5
    from plone.app.multilingual.interfaces import IMutableTG
    from plone.app.multilingual.interfaces import ITranslatable
    ...

else:
    # Plone <= 4
    from plone.multilingual.interfaces import IMutableTG
    from plone.multilingual.interfaces import ITranslatable
    ...
```

This should then work for both versions..